sdk/sync/common.go

```diff
@@ -28,7 +29,7 @@ const (
 //
 // TODO: source redis URL from environment variables. The Redis host and port
 // will be wired in by Nomad/Swarm.
-func redisClient(runenv *runtime.RunEnv) (client *redis.Client, err error) {
+func redisClient(ctx context.Context, runenv *runtime.RunEnv) (client *redis.Client, err error) {
```

Comment:
Let's make clear that this context is only used for resolving the hostname; it in no way governs the lifecycle of the redis client. It's pretty confusing.